pages/[profileSlug]/schemas/[contentSlug]/settings.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import { GetServerSideProps } from "next";
+
+import { SchemaPageFrame, Section } from "components";
+import { SchemaPageHeaderProps } from "components/SchemaPageFrame/SchemaPageFrame";
+// import prisma from "utils/server/prisma";
+import { getSchemaPageHeaderData, getSchemaPagePermissions } from "utils/server/schemaPages";
+
+type SchemaPageParams = {
+	profileSlug: string;
+	contentSlug: string;
+};
+
+interface SchemaSettingsProps {
+	schemaPageHeaderProps: SchemaPageHeaderProps;
+}
+
+export const getServerSideProps: GetServerSideProps<SchemaSettingsProps, SchemaPageParams> = async (
+	context
+) => {
+	const { profileSlug, contentSlug } = context.params!;
+	const schemaPageHeaderProps = await getSchemaPageHeaderData(profileSlug, contentSlug);
+	const hasAccess = getSchemaPagePermissions(context, schemaPageHeaderProps);
```

Comment:
In this case, we probably shouldn't let people view the settings page of schemas that they don't own, even if they're public